src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -158,6 +161,37 @@ export default React.memo(function GamePage(): JSX.Element | null {
                 throw 'Cannot get game info'
               }
               setGameInstallInfo(info)
+              if (
+                runner === 'gog' &&
+                (info?.game?.launch_options || []).length === 0
+              ) {
+                readGOGGameInfoFile(appName)
+                  .then((infoFile) => {
+                    const newLaunchOptions: LaunchOption[] = []
+                    infoFile?.playTasks.forEach((task, index) => {
+                      if (
+                        task.type === 'FileTask' &&
+                        !task?.isHidden &&
+                        task.category !== 'document'
+                      ) {
+                        newLaunchOptions.push({
+                          name: task?.name || infoFile.name,
```

Comment:
Is it possible for a task's `name` property to be undefined?

If this is correct, could you also adapt the `TaskBase` interface?